### PR TITLE
Cleanup the landing page of each utility's documentation

### PR DESCRIPTION
**Pull request by edwardhartnett**
Part of #191 

Landing page cleanup, also add link to the programs on the landing page, so their documentation can be found easily.

### CODE REVIEW

File: sorc/vcoord_gen.fd/docs/user_guide.md [hunk 1]
```diff
@@ -3,16 +3,15 @@
 
 # Introduction
 
-Generates hybrid coordinate parameters from fields such as surface
-pressure, model top and the number of vertical levels. Outputs the
-'ak' and 'bk' parameters used by the forecast model to define the
-hybrid levels.
+The vcoord_gen program driver.F90 generates hybrid coordinate
```
Comment (GeorgeGayno-NOAA) [inline sorc/vcoord_gen.fd/docs/user_guide.md:6]: Do you want to add a link to driver.F90 here?

Comment (GeorgeGayno-NOAA) [reply sorc/vcoord_gen.fd/docs/user_guide.md:6]: I see.  The file extension is lower case - .f90

**Comment by edwardhartnett** [comment]
@GeorgeGayno-NOAA if we can get this reviewed, I'll merge it...

File: sorc/vcoord_gen.fd/docs/user_guide.md (hunk 1 above)
Comment (GeorgeGayno-NOAA) [reply sorc/vcoord_gen.fd/docs/user_guide.md:6]: Sorry @edwardhartnett my comment was not clear.  This link to driver.F90 did not work for me.   The vcoord_gen driver is driver.f90 (with lowercase f90).

Comment (GeorgeGayno-NOAA) [reply sorc/vcoord_gen.fd/docs/user_guide.md:6]: Here is what I see: https://www.emc.ncep.noaa.gov/jcsda/ggayno/doxygen/html/vcoord_gen/index.html

Comment (edwardhartnett) [reply sorc/vcoord_gen.fd/docs/user_guide.md]: OK, good catch, thanks. I fixed...

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Looks good.

**Pull request merged**